src/PrestaShopBundle/Resources/config/services/core/grid/accessibility_checker.yml

```diff
@@ -18,3 +18,6 @@ services:
 
   prestashop.core.grid.action.row.accessibility_checker.delete_order_states:
     class: 'PrestaShop\PrestaShop\Core\Grid\Action\Row\AccessibilityChecker\DeleteOrderStatesAccessibilityChecker'
+
+  PrestaShop\PrestaShop\Core\Grid\Action\Row\AccessibilityChecker\ProductMultiShopPreviewAccessibilityChecker:
+    autowire: true
```

Comment:
public: false maybe?

Reply:
I left it public since all the other were public as well, but why not

Reply:
>> I left it public since all the other were public as well
All the other are in public is not a legit argue :) The other services may be wrong
Considering https://github.com/PrestaShop/ADR/blob/master/0019-FQCN-Autowiring.md as long as it's possible, we should keep services private to optimise the DIC ;)